src/netex_idf/lines.rs

```diff
@@ -42,7 +42,7 @@ const DEFAULT_TEXT_COLOR: Rgb = Rgb {
     blue: 255,
 };
 
-#[derive(Debug)]
+#[derive(Debug, Default)]
```

Comment:
Default or not Default, that is the question :laughing:

Reply:
Oops, I probably pushed part of what I experimented on the side... It should be fixed now.